utils/kext.py

```diff
@@ -41,7 +41,7 @@ def __init__(self,objects,isKernel=False,shared_p = "macOS11.2.3"):
         self.read32 = lambda off: struct.unpack("<I",self.br.readByteArray(off,4))[0]
         self.image_base = currentProgram.getImageBase()
         self.provider = RandomAccessByteProvider(f)
-        self.header = MachHeader.createMachHeader( RethrowContinuesFactory.INSTANCE, self.provider )
+        self.header = MachHeader(self.provider)
```

Comment:
There's a second occurrence of `createMachHeader` in `custom_kc.py`, probably worth fixing too